src/wp-includes/cache-compat.php

```diff
@@ -35,3 +35,86 @@ function wp_cache_get_multiple( $keys, $group = '', $force = false ) {
 		return $values;
 	}
 endif;
+
+if ( ! function_exists( 'wp_cache_delete_multiple' ) ) :
+	/**
+	 * Retrieves multiple values from the cache in one call.
+	 *
+	 * Compat function to mimic wp_cache_delete_multiple().
+	 *
+	 * @ignore
+	 * @since 6.0.0
+	 *
+	 * @see wp_cache_delete_multiple()
+	 *
+	 * @param array  $keys  Array of keys under which the cache to deleted.
+	 * @param string $group Optional. Where the cache contents are grouped. Default empty.
+	 * @return array Array of return values organized into groups.
+	 */
+	function wp_cache_delete_multiple( $keys, $group = '' ) {
```

Comment:
Forcing an `array` here removes the need to do any check on the function and relies on the fact that an array is always provided.

```suggestion
	function wp_cache_delete_multiple( array $keys, $group = '' ) {
```

Reply:
Actioned in https://github.com/WordPress/wordpress-develop/pull/2018